apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -77,6 +78,7 @@ public static Builder builder() {
   private final CacheControl defaultCacheControl;
   private final CacheHeaders defaultCacheHeaders;
   private final ApolloLogger logger;
+  private final ApolloCallTracker callTracker = new ApolloCallTracker();
```

Comment:
I guess we need to make it lazy, so if I don't need to tracker we should avoid overhead of  `ApolloCallTracker ` (it uses synchronization blocks a lot)